src/cmd-buildextend-installer

```diff
@@ -95,6 +95,7 @@ initrd_ignition_padding = 256 * 1024
 def generate_iso():
     arch = platform.machine()
     tmpisofile = os.path.join(tmpdir, iso_name)
+    img_qemu = os.path.join(builddir, buildmeta['images']['qemu']['path'])
```

Comment:
Could probably swap it out for the metal image, since that's more likely to be built alongside the installer ISO vs. the qemu image.

(Or another more heavyweight approach is to refactor things so that we can get the same list of kargs that would be used even if we didn't create a metal/qemu image at all.)

Reply:
Right, although I'd rather have this reworked in a large-scale refactor separately